### PR TITLE
Fix AtomicConv slow tests

### PR DESCRIPTION
**Pull request by ncfrey**
## Description

Partial fix for #2403 

I have cleaned up `AtomicConvModel` and fixed the slow tests. I used `pytest timeout` for some extensive testing and found that, at least on my local machine, the featurization and fitting was timing out, which was causing test failures. I reduced the size of the dense layers in the models for the tests, which speeds them up considerably, and allows for testing.

The tests are still "slow" (taking about 60 seconds on my machine), so I added some simple tests for initializing `AtomicConvModel` so that we don't have to rely completely on the slow tests, which are only run occasionally. I also added some additional tests to make sure that the trained models are not outputting all zeros for predictions.

## Type of change

Please check the option that is related to your PR.

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: deepchem/models/atomic_conv.py
Comment (rbharath) [inline deepchem/models/atomic_conv.py:180]: Should we go ahead and remove some of the commented lines here and elsewhere in this PR?

File: deepchem/models/atomic_conv.py
Comment (rbharath) [inline deepchem/models/atomic_conv.py:181]: One point worth noting here is that this PR (and my earlier PR) both make a major change to the `AtomicConv` architecture by cutting off `AtomicConvScore` and adding a stack of dense layers at the end. This is a little less physically inspired but perhaps more robust in practice.

For us to really have a sense for whether this architecture works, we'll need to run some benchmarks on PDBBind and get numbers. This is a larger project though! Might be a good target for future PRs

**Review by rbharath — APPROVED** [review]
This is looking good! Have some minor cosmetic comments but I think you can go ahead and merge this in once you're ready :)

File: deepchem/models/atomic_conv.py
Comment (ncfrey) [reply deepchem/models/atomic_conv.py:180]: Sounds good! I'll go ahead and remove the unnecessary comments.

File: deepchem/models/atomic_conv.py
Comment (ncfrey) [reply deepchem/models/atomic_conv.py:181]: Yes, this is a big change - once this is merged in I'll start working on the tutorial and do some preliminary testing to see what the scope of a larger benchmark might look like.

**Comment by ncfrey** [comment]
I deleted more debugging print statements and cleared out some old commented-out code. Can merge this in whenever and get started on the tutorial!

**Comment by rbharath** [comment]
Awesome :). Let's merge this in once tests finish running

**Comment by ncfrey** [comment]
Tests look good, going to go ahead and merge in.

**Pull request merged**
